From 15cf1a31f5af8f09531bb837b92bd6ea49bd1744 Mon Sep 17 00:00:00 2001
From: Pascal Bach <pascal.bach@siemens.com>
Date: Wed, 13 Sep 2017 08:41:21 +0200
Subject: [PATCH] Allow newer versions of requests

docker compose has strict requirements to use requests < 2.12

However it works without issues with newer versions, so this patch removes the check.

Upstream-Status: Pending

Signed-off-by: Pascal Bach <pascal.bach@siemens.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 192a0f6..f444757 100644
--- a/setup.py
+++ b/setup.py
@@ -33,7 +33,7 @@ install_requires = [
     'cached-property >= 1.2.0, < 2',
     'docopt >= 0.6.1, < 0.7',
     'PyYAML >= 3.10, < 4',
-    'requests >= 2.6.1, != 2.11.0, < 2.12',
+    'requests >= 2.6.1, != 2.11.0',
     'texttable >= 0.9.0, < 0.10',
     'websocket-client >= 0.32.0, < 1.0',
     'docker >= 2.5.1, < 3.0',
-- 
2.1.4

